tests/conftest.py

```diff
@@ -29,7 +29,7 @@ def pytest_sessionstart(session):
     Project.set_repo_path(TEST_REPO_PATH)
 
     _init_test_repo()
-    os.environ[AIM_WEB_ENV_KEY] = 'test'
+    os.environ[AIM_ENV_MODE_KEY] = 'test'
```

Comment:
I think we should set environment before `_init_test_repo()` to make sure run metadata db log is captured.

Reply:
done ✅